src/axom/inlet/tests/inlet_object.cpp

```diff
@@ -802,6 +802,58 @@ TYPED_TEST(inlet_object, primitive_arrays_as_std_vector)
   EXPECT_EQ(arr_w_indices, expected_arr_w_indices);
 }
 
+TYPED_TEST(inlet_object, primitive_arrays_as_std_vector_wrong_type)
+{
+  std::string testString = " arr = { [0] = 'a', [1] = 'b', [2] = 'c'}";
+  DataStore ds;
+  Inlet inlet = createBasicInlet<TypeParam>(&ds, testString);
+
+  // Define schema
+  inlet.addIntArray("arr");
+
+  // The array was empty (same as if it didn't exist), but *not* marked as required
+  EXPECT_TRUE(inlet.verify());
```

Comment:
The plan is to correct this to a `EXPECT_FALSE` in a future MR, correct? For now, this test will just verify the code won't crash.

Reply:
Yes - in the follow-up, the tests will be as follows:

https://github.com/LLNL/axom/blob/7435bba50ea921d145aa48a02c4bfd8013030f8e/src/axom/inlet/tests/inlet_object.cpp#L805-L844